src/db/migrations/migration2.js

```diff
@@ -0,0 +1,33 @@
+const { getLogger } = require('../../utils/logger');
+const { map } = require('lodash');
+
+async function addTypeField(db) {
+  try {
+    // await db.update({ type: { $exists: false }, token: 'QTUM' }, { $set: { type: 'BET' } }, { multi: true });
+    // await db.update({ type: { $exists: false }, token: 'QTUM' }, { $set: { type: 'BET' } }, { multi: true });
+    const votes = await db.Votes.find({ type: { $exists: false } });
+    map(votes, async (vote) => {
+      if (vote.token === 'QTUM') {
+        await db.Votes.update({ txid: vote.txid }, { $set: { type: 'BET' } });
```

Comment:
create a var for `type` and call `await db.Votes.update` once at the end of the if checks